lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java

```diff
@@ -7121,6 +7122,7 @@ private long importCcFromStream(InputStream fis) {
 			    topicobject = q;
 		    }
 
+            log.debug("parser.parse {} {} {} {} {} {} {}", this, cartridgeLoader, simplePageToolDao, quizobject, topicobject, bltiEntity, assignobject);
```

Comment:
Spacing

Reply:
The spacing on these lines includes a mix of tabs and spaces - sigh.   I will fix the spacing by matching the current spacing.